Gemfile.lock

```diff
@@ -732,9 +732,3 @@ DEPENDENCIES
   whenever
   wicked_pdf
   wkhtmltopdf-binary
-
-RUBY VERSION
-   ruby 2.1.5p273
-
-BUNDLED WITH
-   1.13.6
```

Comment:
IMHO would be safer to keep `Gemfile.lock` as it is and make sure two things:

 - that every contributor uses the latest `bundler` version
 - that every contributor uses the exactly same ruby version that we have in production
 - that every instance of OFN uses the same ruby version (most difficult part IMHO)

This strategy would avoid future headaches and make deploy process more consistent and stable.